README.md

```diff
@@ -166,6 +166,14 @@ If you want to limit memory to 64GB but allow MarkLogic Server to use swap space
 ```
 To allow MarkLogic Server to use unlimited memory and swap space (up to the amount available on the host system), do not specify either "--memory" or "--memory-swap".
 
+## Configuring HugePages
+
+By default, if HugePages are configured on the host, MarkLogic running in a container will attempt to allocate up to 3/8 of the container memory limit as HugePages. For example, consider a host with 128GB of RAM, 48GB HugePages, and running two MarkLogic containers, each with 64GB memory limit. MarkLogic in each container will only allocate up to 24GB in HugePages (3/8 * 64GB).
```

Comment:
By default, if HugePages are configured on the host, the MarkLogic instance running in a container will attempt to allocate up to 3/8 of the container memory limit as HugePages. For example, consider a host with 128GB of RAM, 48GB HugePages, and running two MarkLogic containers, each with 64GB memory limit. The MarkLogic instance in each container will only allocate up to 24GB in HugePages (3/8 * 64GB).

(the MarkLogic instance)

Reply:
Updated